.cicd/platforms/asan.Dockerfile

```diff
@@ -19,17 +19,17 @@ RUN apt-get update && apt-get upgrade -y && \
                        zlib1g-dev           \
                        zstd
 
-RUN yes | bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)" llvm.sh 19
+RUN yes | bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)" llvm.sh 20
```

Comment:
Maybe you can make the version number (`20`) a variable. Next time you only change one place. But this is not a big deal.